webdriver-spec.html

```diff
@@ -742,6 +742,9 @@
 
   <p>Otherwise, let <var>parameters</var> be null.</p>
 
+ <li><p>If a <a>navigation</a> is currently underway <a>wait for
+  navigation to complete</a>.
```

Comment:
The intent is to match what we do on "`Go`": use the current `page loading strategy` to determine when to process the next command. This is the (admittedly not clearly documented) behaviour of the Selenium WebDrivers.